playbooks/deploy-osp.yml

```diff
@@ -178,12 +191,7 @@
       become_user: stack
 
     - name: Overcloud deploy - tail -f /home/stack/logs/overcloud_install.log to view progress
-      shell: |
-        set -o pipefail
-        source /home/stack/stackrc
-        openstack overcloud deploy --templates \
-        -e /home/stack/templates/node-info.yaml \
-        -e /home/stack/templates/overcloud_images.yaml \
-        --ntp-server pool.ntp.org \
-        2>&1 | tee /home/stack/logs/overcloud_install.log
+      shell: sh overcloud-deploy.sh
```

Comment:
script has shebang of `bash`, but here we call it with `sh` (which can be two different shells or two different behaviors from the same shell).